src/dmd/dsymbolsem.d

```diff
@@ -1215,9 +1215,21 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
                 }
                 else
                 {
+                    import dmd.hdrgen : toCBuffer, HdrGenState;
+                    OutBuffer buff;
+                    HdrGenState hgs;
+                    toCBuffer(dsym, &buff, &hgs);
+                    if (buff.offset != 0)
+                        buff.setsize(buff.offset-1);
+
                     // https://issues.dlang.org/show_bug.cgi?id=14166
                     // Don't run CTFE for the temporary variables inside typeof
                     dsym._init = dsym._init.initializerSemantic(sc, dsym.type, sc.intypeof == 1 ? INITnointerpret : INITinterpret);
+                    const init_err = dsym._init.isExpInitializer();
+                    if (init_err && init_err.exp.op == TOK.errExtraInfo)
+                    {
+                         errorSupplemental(dsym.loc, "compile time context created here: `%s`", buff.extractString());
```

Comment:
Looks like `buff.extractString()` contains a trailing newline on Windows, for some reason.